sql/mozfun/datetime_util/braze_parse_time/udf.sql

```diff
@@ -0,0 +1,12 @@
+CREATE OR REPLACE FUNCTION datetime_util.braze_parse_time(json_str STRING)
+RETURNS STRING
+LANGUAGE js
+AS
+  """
+  try {
```

Comment:
Looks like this could be easily done via stardard SQL in BQ:
https://cloud.google.com/bigquery/docs/reference/standard-sql/json_functions#json_extract
?

Reply:
Does `JSON_VALUE(json_str, '$."$time"')` work?  `SELECT JSON_VALUE('{"$time":"2024-05-05 14:46:02.000000 UTC"}', '$."$time"')` seems to work